tests/conftest.py

```diff
@@ -12,18 +12,18 @@
 
 @pytest.fixture()
 def config():
-    with open("config.json") as config_file:
+    with open("../config.json") as config_file:
```

Comment:
Dodaj w biblioteke `from pathlib import Path`. Generalnie powinno się tą biblioteką obsługiwać ścieżki. Dzięki temu są one niezależne od systemu operacyjnego, gdy tworzy się obiekt `Path(sciezka)`
```suggestion
    with open(Path("./config.json")) as config_file:
```